### PR TITLE
Update categories_map.py

### PR DESCRIPTION
**Pull request by perrotta**
Following PR  #29137:

- The following package is assigned to AlCa
"CondFormats/PPSObjects,"  -> it replaces "CondFormats/CTPPSReadoutObjects"

- The following packages are assigned to reco
   "RecoPPS/Configuration"    -> it replaces "RecoCTPPS/Configuration"
    "RecoPPS/PixelLocal"           -> it replaces  "RecoCTPPS/PixelLocal"
    "RecoPPS/ProtonReconstruction"  -> it replaces  "RecoCTPPS/ProtonReconstruction"

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: categories_map.py [hunk 1]
```diff
@@ -1292,6 +1293,9 @@
     "RecoPixelVertexing/PixelTrackFitting",
     "RecoPixelVertexing/PixelTriplets",
     "RecoPixelVertexing/PixelVertexFinding",
+    "RecoPPS/Configuration",
+    "RecoPPS/PixelLocal",
```
Comment (smuzaffar) [inline categories_map.py]: @perrotta , I guss this should be `RecoPPS/Local` correct?

Comment (perrotta) [reply categories_map.py]: Ops, yes. My fault sorry.
Thank you for noticing @smuzaffar : I prepare a fix

Comment (perrotta) [reply categories_map.py]: See https://github.com/cms-sw/cms-bot/pull/1272

**Pull request merged**